roles/cifmw_external_dns/tasks/get_dns_forward_index.yml

```diff
@@ -0,0 +1,78 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Get current DNS configuration
+  delegate_to: localhost
```

Comment:
do we need to run this same for each compute? Else, `run_once: true` might speed things up

Reply:
I'll add the `run_once: true`.